site/server/devServer.tsx

```diff
@@ -122,10 +123,25 @@ devServer.get(`/${covidDashboardSlug}`, async (req, res) => {
 devServer.get(
     `/${covidCountryProfileRootPath}/:countrySlug`,
     async (req, res) => {
-        res.send(await covidCountryProfileCountryPage(req.params.countrySlug))
+        res.send(
+            await countryProfileCountryPage(
+                countryProfileSpecs.find(
```

Comment:
nit: should you store this in a var like you do below for `co2profile` (or vice versa?)